reactor-core/src/main/java/reactor/core/publisher/FluxReplay.java

```diff
@@ -1598,7 +1606,7 @@ static long markDisposed(ReplaySubscriber<?> instance) {
 		}
 
 		/**
-		 * Check if state has subscribed flag indicating subscription reception
+		 * Check if state has {@link #CONNECTED_FLAG} flag indicating subscription reception
```

Comment:
which of SUBSCRIBED_FLAG and CONNECTED_FLAG indicates `subscription reception`? both this javadoc and the one below now correctly link to the right constant, but they otherwise use the exact same phrasing.

Reply:
fixed